Samples/WinMLSamplesGallery/.gitignore

```diff
@@ -1,3 +1,4 @@
 *.onnx
 !squeezenet1.1-7-batched.onnx
```

Comment:
don't we need these onnx files? where do they come from if git ignores them?

Reply:
All of the models covered by *.onnx are configured to be downloaded from the onnx model zoo. The only ones that aren't from the onnx model zoo are ones that we've modified and need to be shipped with, such as the squeeze net model (I think numfor manually set a batch size override?)